cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceHandler.java

```diff
@@ -97,8 +99,7 @@ public void list(HttpRequest request, HttpResponder responder,
    *
    * @param namespaceId namespace of the dataset instance
    * @param name name of the dataset instance
-   * @param owners a list of owners of the dataset instance, in the form @{code <type>::<id>}
-   *               (e.g. "program::namespace:default/application:PurchaseHistory/program:flow:PurchaseFlow")
+   * @param owners a UTF-8-encoded JSON list of Id owners of the dataset instance
```

Comment:
Is this a public REST API? If so, we need to keep the old syntax and deprecate it